mpp/build.gradle.kts

```diff
@@ -131,6 +131,7 @@ tasks.register("testUIKit") {
     dependsOn(":compose:ui:ui-text:$subtaskName")
     dependsOn(":compose:ui:ui:$subtaskName")
     dependsOn(":compose:material3:material3:$subtaskName")
+    dependsOn(":compose:foundation:foundation:$subtaskName")
```

Comment:
Why is it needed? It should be already available implicitly as dependency of material

Reply:
It is not a maven dependency, it is a Gradle Task "testUIKit"